src/Esfa.Vacancy.Domain/Validation/ErrorMessages.cs

```diff
@@ -45,6 +45,9 @@ public static class CreateApprenticeship
             public const string ExpectedStartDateBeforeClosingDate = "'Expected Start Date' must be after the specified application closing date.";
             public const string CreateApprenticeshipParametersIsNull = "Either no request was provided or the request could not be recognised.";
             public const string MissingProviderSiteEmployerLink = "User entry is invalid or no existing link between the provider site and employer.";
+
+            public const string LocationFieldNotRequired =
+                "'{PropertyName}' is not required when Location type is EmployerLocation or Nationwide";
```

Comment:
'{PropertyName}' *can't be specified* when Location type is EmployerLocation or Nationwide

^^ because this is a validation error if we say 'is not required' it sounds it should still be allowed.

Reply:
Fixed